akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -73,36 +76,14 @@ private[akka] class PublisherSink[In](val attributes: Attributes, shape: SinkSha
    * subscription a VirtualProcessor would perform (and it also saves overhead).
    */
   override def create(context: MaterializationContext): (AnyRef, Publisher[In]) = {
-    val proc = new VirtualPublisher[In]
+    val proc = new VirtualPublisher[In]()
```

Comment:
Per my understanding we need parenthesis if there is some side effect (this is as per http://docs.scala-lang.org/style/method-invocation.html) Still we have keyword "new" here that indicates side effect. So let me know if this is still considering good style for the project